<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1385</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/lynxLogoScaled.png</normaloff>:/images/lynxLogoScaled.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="VideoWidget" name="cameraWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="MapAndRender" name="mapAndRenderWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>461</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="ChartWidget" name="lynxChartWidget" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>901</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>1361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>31</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>88</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <column>
     <property name="text">
      <string>TAKIM</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PAKET</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ZAMAN</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>BASINÇ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>YÜKSEKLİK</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>İNİŞ HIZI</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>SICAKLIK</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PİL</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>LATITUDE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>LONGITUDE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ALTITUDE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DURUM</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PICH</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ROLL</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>YAW</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DÖNÜŞ SAYISI</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>VİDEO AKTARMA</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="sendVideoFileButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>510</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Video Gönder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendLeaveCommandButton">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>510</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ayrıl</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DATE :  TEAM NO:#55066</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>510</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bağlan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reconnectButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>510</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Yeniden Bağlan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>510</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bağlantı Kes</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="sendVideoFileProgressBar">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>510</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>510</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SENDING DATA</string>
    </property>
   </widget>
   <widget class="QComboBox" name="serialPortComboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>510</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateMotorButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>510</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor Döndür</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openMotorButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Kol Aç</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeMotorButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Kol Kap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mergeButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>510</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Birleş</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1385</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">videowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MapAndRender</class>
   <extends>QWidget</extends>
   <header location="global">mapandrender.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChartWidget</class>
   <extends>QWidget</extends>
   <header location="global">chartwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>sendVideoFileButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sendVideoFileClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>930</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>710</x>
     <y>552</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>connectButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reconnectButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>reconnectButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>disconnectButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>disconnectButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>730</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>706</x>
     <y>465</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendLeaveCommandButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sendLeaveCommandButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1240</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>828</x>
     <y>685</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openMotorButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openMotorClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeMotorButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>closeMotorClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>634</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>752</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mergeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mergeButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>885</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>915</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotateMotorButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>rotateMotorClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>666</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sendVideoFileClick()</slot>
  <slot>connectButtonClick()</slot>
  <slot>reconnectButtonClick()</slot>
  <slot>disconnectButtonClick()</slot>
  <slot>sendLeaveCommandButtonClick()</slot>
  <slot>openMotorClick()</slot>
  <slot>closeMotorClick()</slot>
  <slot>mergeButtonClick()</slot>
  <slot>rotateMotorClick()</slot>
 </slots>
</ui>
